portfolio/src/main/webapp/script.js

```diff
@@ -40,4 +41,77 @@ function showRandomQuotes() {
   // Add it to the page.
   const quotesContainer = document.getElementById('greeting-container');
   quotesContainer.innerText = quote;
+}
+/**
+ * Shows comments on the page.
+ */
+function showComments() {
+  hideComments();
+  fetch('/data').then(response => response.json()).then((comments) => {
+    const commentsListElement = document.getElementById('comments-container');
+    comments.forEach(comment => {
+      commentsListElement.appendChild(
+      createListElement(comment));
+    });
+  });
+}
+/**
+ * Creates a list of comments and displays it to main page.
+ * @params {string} text
+ * @return {!Element}
+ */
+function createListElement(user) {
+  const liElement = document.createElement('li');
+  liElement.innerText = user.emailId + " : " + user.comment;
+  return liElement;
+}
+
+/**
+ * Hides comments from the main page.
+ */
+function hideComments() {
+  const list = document.getElementById('comments-container');
+  list.innerText = "";
+}
+
+/**
+ * Checks whether the user is logged in or not.
+ */
+function checkLoginInfo() {
+  fetch('/login').then(response => response.json()).then(userLogin => {
+    const checkLoginInfo = document.getElementById('checklogininfo-container');
```

Comment:
Since these are HTML elements, its better to name them accordingly.

```suggestion
    const checkLoginInfoElement = document.getElementById('checklogininfo-container');
```

Same for others